src/Styles/Thickness.xaml

```diff
@@ -38,4 +38,6 @@
 
     <Thickness x:Key="SettingsCardMargin">0,0,0,4</Thickness>
     <x:Double x:Key="SettingsCardSpacing">4</x:Double>
+

```

Comment:
NIT: Remove extra line.

Reply:
The extra line is expected because it's for a different page.